pilot/pkg/serviceregistry/kube/controller/network_test.go

```diff
@@ -237,6 +238,20 @@ func TestSyncAllWorkloadsFromAmbient(t *testing.T) {
 
 	s := newAmbientTestServer(t, testC, "")
 
+	lastNotified := atomic.Value{}
+	s.controller.namespaces.AddEventHandler(controllers.ObjectHandler(func(o controllers.Object) {
+		lastNotified.Store(o.GetName())
```

Comment:
can you do this notification via a channel

Reply:
Updated.